Mage.Sets/src/mage/cards/a/AgainstAllOdds.java

```diff
@@ -0,0 +1,65 @@
+package mage.cards.a;
+
+import mage.abilities.Mode;
+import mage.abilities.effects.common.ExileTargetForSourceEffect;
+import mage.abilities.effects.common.ReturnFromGraveyardToBattlefieldTargetEffect;
+import mage.abilities.effects.common.ReturnToBattlefieldUnderOwnerControlTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.filter.FilterCard;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.target.common.TargetControlledPermanent;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class AgainstAllOdds extends CardImpl {
+    private static final FilterControlledPermanent filter = new FilterControlledPermanent("creature card with mana value 3 or less");
```

Comment:
this filter shouldn't be mana value 3 or less

Reply:
fixed